WooCommerce/Classes/Model/CustomHelpCenterContent.swift

```diff
@@ -27,8 +27,10 @@ extension CustomHelpCenterContent {
     ///
     init?(step: AuthenticatorAnalyticsTracker.Step, flow: AuthenticatorAnalyticsTracker.Flow) {
         switch step {
-        case .start where flow == .loginWithSiteAddress:
+        case .start where flow == .loginWithSiteAddress: // Enter Store Address screen
             url = WooConstants.URLs.helpCenterForEnterStoreAddress.asURL()
+        case .enterEmailAddress where flow == .loginWithSiteAddress: // Enter WordPress.com email screen
```

Comment:
super nit: might be worth mentioning the site address flow to differentiate it from the WP.com flow
```suggestion
        case .enterEmailAddress where flow == .loginWithSiteAddress: // Enter WordPress.com email screen from store address flow
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/7573/commits/27fd7007e2dcaf1e3a5ace1467b31264ac326c28